samples/test-samples.cmake

```diff
@@ -64,45 +83,50 @@ foreach (SAMPLE ${SAMPLES_LIST})
       message(STATUS "Samples test '${SAMPLE}' with CMake passed!")
     endif ()
 
-    # Build with pkg-config
-    message(STATUS "Samples test '${SAMPLE}' with pkg-config running...")
-    execute_process(
-      COMMAND ${CMAKE_COMMAND} -E env PATH=${INSTALL_DIR}/bin:$ENV{PATH} PKG_CONFIG_PATH=${INSTALL_DIR}/share/pkgconfig/ make -C ${SAMPLE_SOURCE_DIR} clean build run
-      RESULT_VARIABLE TEST_RESULT)
-      if (TEST_RESULT)
-        message(WARNING "Samples test '${SAMPLE}' with pkg-config failed!")
+    if (NOT WIN32)
```

Comment:
Should this be `if (UNIX)` instead? Here and below on line 102 ... (i.e. is that the only dichotomy of OS types we care about?)

Reply:
Good question. I think we mostly choose to use `if (UNIX)` in our cmake files, so I should keep it consistent